dataframe/src/main/groovy/com/elintegro/erf/widget/vue/GridWidgetVue.groovy

```diff
@@ -221,10 +260,47 @@ $fieldParams
 
         String dataVariable = dataframe.getDataVariableForVue(fieldProps)
         def dataHeader = fieldProps.dataHeader
+        boolean saveEditedFieldData = fieldProps.saveEditedFieldData?:false
+        String tableName = fieldProps.tableName
         String headerString = "${getFieldJSONNameVue(fieldProps)}${DOT}${headers}"
         dataframe.getVueJsBuilder().addToMethodScript(""" getDefaultDataHeaders_${dataVariable} : function(){\n
                              var defaultDataHeaders = ${dataHeader as JSON};
                              this.$headerString = defaultDataHeaders;
+                          },\n
+                          save (data) {
+                              var params = this.state;
+                              params['id'] = data.Id;
+                              params['dataOfSelectedRow'] = data;
+                              params['editableField'] = '$fieldProps.editableField'
+                              params['tableName'] = '$tableName';
+                              params['dataframe'] = '$dataframe.dataframeName'
+                              var self = this;
+                              if($saveEditedFieldData == true){
+                                      excon.callApi('$gridSaveUrl', 'post', params).then(function(responseData){
+                                                 var response = responseData.data;
+                                                 self.snack = true
+                                                 self.snackColor = 'success'
+                                                 self.snackText = 'Data saved'
```

Comment:
Probably, we do have an existing snack bar for message display. Use that one instead.